src/main/scala/sbt/errorssummary/ConciseReporter.scala

```diff
@@ -64,6 +64,11 @@ private class ConciseReporter(logger: Logger,
             log(line)
           }
       }
+
+    logger.info("Legend:")
+    logger.info("Ln: line number n")
+    logger.info("Cn: column number n")
+    logger.info("En: error number n")
```

Comment:
Sure, I can make the legend a single line and add a setting. If you don't mind, perhaps we can design the enhanced (coloured) legend as a future issue/PR?